app/views/departments/_form.html.slim

```diff
@@ -0,0 +1,15 @@
+= simple_form_for(@department) do |f|
+  - if f.error_notification.present?
+    .row
+      .col-xs-12
+        = f.error_notification
+  .row
+    .col-xs-12
+      = f.association :user, as: :select, collection: User.department_assocable
+
+  = f.simple_fields_for :contact do |c|
+    = render 'contact_fields', f: c
+
+  .row
+    .col-xs-12
+      = f.button :submit
```

Comment:
True. I colored it now, but also felt like changing the New email and New phone buttons to small ones and with Add email and Add Phone text, because this makes more sense.
Otherwise there would have been three similar looking big green buttons on top of each other